netmiko/cisco/cisco_bsp.py

```diff
@@ -0,0 +1,284 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+from logger.cafylog import CafyLog
+from netmiko.cisco_base_connection import CiscoBaseConnection
+from netmiko.ssh_exception import NetMikoAuthenticationException
+import re
+import time
+
+log = CafyLog()
+
+class CiscoBsp(CiscoBaseConnection):
+    '''
+    CiscoBsp is based of CiscoBaseConnection
+    '''
+    def bmc_to_bsp_prompt(self,bmc_prompt_pattern=r'\s*\#',bsp_prompt_pattern=r']\s*\#',delay_factor=1,max_loops=20):
+        '''
+        Switch from BMC to BSP prompt
+
+        :param bmc_prompt_pattern: expected BMC prompt
+        :param bsp_prompt_pattern: expected BSP prompt
+        :param delay_factor: Factor to adjust delays
+        :param max_loops: max number of iterations for attempting to switch prompts
+        :return:
+        '''
+
+        self.TELNET_RETURN = '\n'
+        bmc_to_bsp_cmd = "sol.sh"
```

Comment:
please put the full path as in /usr/xxx

Reply:
bmc_to_bsp_cmd = "/usr/local/bin/sol.sh"